sycl/unittests/pi/EnqueueMemTest.cpp

```diff
@@ -109,7 +109,7 @@ INSTANTIATE_TEST_SUITE_P(
     EnqueueMemTestImpl, EnqueueMemTest,
     testing::ValuesIn(pi::initializeAndRemoveInvalid()),
     [](const testing::TestParamInfo<EnqueueMemTest::ParamType> &info) {
-      return pi::GetBackendString(info.param);
+      return pi::GetBackendString(info.param).c_str();
```

Comment:
Unfortunately, this also causes "error: returning address of local temporary object"